packages/runtime/client/rest/src/contract/transaction/details.ts

```diff
@@ -1,37 +1,116 @@
 import { optionFromNullable } from "io-ts-types";
 import * as t from "io-ts/lib/index.js";
+import * as O from "fp-ts/lib/Option.js";
 
 import { ISO8601 } from "@marlowe.io/adapter/time";
-
 import * as G from "@marlowe.io/language-core-v1/guards";
+import {
+  BuiltinByteString,
+  Contract,
+  Input,
+  MarloweState,
+} from "@marlowe.io/language-core-v1";
 import {
   TagsGuard,
   Metadata,
   BlockHeaderGuard,
   TxOutRef,
   TxId,
   TextEnvelopeGuard,
+  ContractId,
+  Tags,
+  BlockHeader,
+  TextEnvelope,
 } from "@marlowe.io/runtime-core";
 
 import { ContractIdGuard } from "@marlowe.io/runtime-core";
 import { TxStatus } from "./status.js";
+import { assertGuardEqual, proxy } from "@marlowe.io/adapter/io-ts";
+
+/**
+ * Represents the response of the {@link index.RestAPI#getContractTransactionById | Get contract transaction by ID } endpoint
+ *
+ * @experimental
+ * This type is experimental and subject to change. See current DISCUSSIONS in the source code.
+ */
```

Comment:
cc @nhenin , we should probably add this note to any experimental endpoint or attribute